charts/vault/templates/vault-rb.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.plugins.readKuberenetsSecrets.namespaces }}
+{{- $releaseNamespace := .Release.Namespace -}}
```

Comment:
Consider using "{{ $.Release.Namespace }}" directly instead of adding this variable

Reply:
it did not work for some reason... that is why I used this variable here

Reply:
Changed that. Thanks